### PR TITLE
Runtime: switch default case come last

### PR DESCRIPTION
**Pull request by hhugo**
It makes the code a bit easier to read
@vouillon, can you review this ?

### CODE REVIEW

**Comment by hhugo** [comment]
cc @smorimoto

**Review by smorimoto — APPROVED** [review]

**Comment by hhugo** [comment]
@smorimoto, I would have preferred a review from @vouillon before merging.

**Comment by smorimoto** [comment]
Oops, sorry about that.

**Pull request merged**
